cmd/cosign/cli/options/attest_blob.go

```diff
@@ -85,6 +86,10 @@ func (o *AttestBlobOptions) AddFlags(cmd *cobra.Command) {
 		"write everything required to verify the blob to a FILE")
 	_ = cmd.Flags().SetAnnotation("bundle", cobra.BashCompFilenameExt, []string{})
 
+	// TODO: have this default to true as a breaking change
+	cmd.Flags().BoolVar(&o.ProtobufBundleFormat, "protobuf-bundle-format", false,
```

Comment:
Suggestion -

"protobuf" is a data format that is used internally to define various sigstore data structures. If cosign was a black box, then "protobuf" is not actually meaningful to the end user, and in fact the output bundle is formatted as JSON, not a protobuf. The fact that we're internally aligning with an abstract data structure that happens to be defined in a `.proto` file doesn't help the user understand the difference. I think it would be more useful to describe it as the "sigstore bundle", or the "v1 bundle" (or v2?), or to maybe have the flag be something like "new-bundle-format" or "experimental-bundle-format", or some other differentiator that doesn't refer to the underlying implementation details that a user shouldn't need to care about.

Reply:
Sounds good to me! I went with `new-bundle-format`, as I think that's more accurate than experimental (it's the bundle format most of the Sigstore client libraries are using).